### PR TITLE
Estonian translations

### PR DESCRIPTION
**Pull request by reosarevok**
We've added Estonian translations for a fair amount of the descriptions the bot deals with. Additionally I added two extra descriptions I'd love the bot to translate, "violinist" and "cellist" (I'd like to translate all sorts of musical descriptions but that's a start).

Hemos añadido traducciones al estonio a la mayoría de las descripciones. Además añadí dos descripciones más que querría hacer que el bot traduzca, "violinista" y "violonchelista" (me gustaría traducir todo tipo de descripciones musicales, pero parece un buen comienzo)

### CODE REVIEW

File: human.descriptions.py [hunk 1]
```diff
@@ -3474,11 +3761,28 @@ def main():
             'bn': { 'male': '~ মডেল', 'female': '~ মডেল' },
             'en': { 'male': '~ model', 'female': '~ model' },
             'es': { 'male': 'modelo ~', 'female': 'modelo ~' },
+            'et': { 'male': '~ modell', 'female': '~ modell' },
             'fr': { 'male': 'mannequin ~', 'female': 'mannequin ~' },
             'he': { 'male': 'דוגמן ~', 'female': 'דוגמנית ~' },
-            'es': { 'male': 'model ~', 'female': 'model ~' },
+            'ro': { 'male': 'model ~', 'female': 'model ~' },
```
Comment (reosarevok) [inline human.descriptions.py]: I'm fairly sure this second "es" was supposed to be "ro" - I made sure that Romanian uses "model" before making this change

**Comment by emijrp** [comment]
@reosarevok Great, thanks! But you haven't added nationalities in Estonian? I can keep this open, so you can work on it, and I will merge when all is done.

**Comment by reosarevok** [comment]
I did, in the second commit :) Is it not showing to you? "+            'et': { 'male': 'Afganistani', 'female': 'Afganistani' }," etc.

**Comment by emijrp** [comment]
@reosarevok Ah, ok! Merged. Wanna launch the bot yourself so you learn to use it?

**Comment by reosarevok** [comment]
If you want :) I have never tried a Wikidata bot yet, but there's always a first time

**Comment by reosarevok** [comment]
Do you have any guide or tutorial to launch the bot? :)

**Comment by emijrp** [comment]
@reosarevok Have you ever used pywikibot? You just have to move the script into the directory and launch it with "python human-descriptions.py"

**Comment by reosarevok** [comment]
Hmm. Wouldn't I need to have an accepted bot account to be allowed to do this? I don't have one :( (I could just use my personal account but that seems to go against the rules)

**Comment by emijrp** [comment]
@reosarevok Yes, you need one. But the approval process is easy and quick.

**Pull request merged**